storybookjs/storybook
# Core: Allow overriding WebView and UrlStore

Issue: React Native wants to override these with non-web versions.

## What I did

1. Allow passing the view+store into `PreviewWeb`
2. Create a super class with the properties we need
3. Type everything

## How to test

Everything should just pass as before in SB proper.

@shilman I am labelling this as `patch` because we want to backport to 6.5.x. However it may not apply cleanly due to the type parameterization. We could do a simpler version that doesn't worry about that?
